plugins/emmc/mmc.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License v2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 021110-1307, USA.
+ *
+ * Modified to add field firmware update support,
+ * those modifications are Copyright (c) 2016 SanDisk Corp.
+ */
```

Comment:
Can we source this header from the kernel somehow?

Reply:
It comes from the mmc-utils userspace package

Reply:
That doesn't export a devel header, does it?

Reply:
No, it doesn't.  So the options are either carry it verbatim or just pull out the `#define`s needed.

Reply:
unless we need more than about 50% of those, I'm erring on just pulling the ones we need.

Reply:
Seems that we need about 20%, I pulled them in.